internal/expr/bindinputs.go

```diff
@@ -155,6 +154,61 @@ type typedOutputExpr struct {
 	outputColumns []outputColumn
 }
 
+type sqlBuilder struct {
+	buf bytes.Buffer
+}
+
+// writeInsert writes the SQL for INSERT statements to the sqlBuilder.
+func (b *sqlBuilder) writeInsert(inputCount int, columns []string) {
+	// Write out the columns.
+	b.buf.WriteString("(")
+	for i, col := range columns {
+		if i != 0 {
```

Comment:
This repeated block could be a method accepting the string to write (`col` in this case).

Reply:
Added a `sqlBuilder.writeCommaSeperatedList` which accepts a writer function for printing each list element to the buffer.